### PR TITLE
 Add caters to PWA on SPA page

### PR DESCRIPTION
**Pull request by peterhashair**
### Description:

 Add caters to PWA to SPA

Follow the request here https://github.com/matomo-org/matomo/issues/18066

### Review

* [ ] [Functional review done](https://developer.matomo.org/guides/pull-request-reviews#functional-review-done)
* [ ] [Potential edge cases thought about](https://developer.matomo.org/guides/pull-request-reviews#potential-edge-cases-thought-about) (behavior of the code with strange input, with strange internal state or possible interactions with other Matomo subsystems)
* [ ] [Usability review done](https://developer.matomo.org/guides/pull-request-reviews#usability-review-done) (is anything maybe unclear or think about anything that would cause people to reach out to support)
* [ ] [Security review done](https://developer.matomo.org/guides/security-in-piwik#checklist)
* [ ] [Wording review done](https://developer.matomo.org/guides/pull-request-reviews#translations-wording-review-done)
* [ ] [Code review done](https://developer.matomo.org/guides/pull-request-reviews#code-review-done)
* [ ] [Tests were added if useful/possible](https://developer.matomo.org/guides/pull-request-reviews#tests-were-added-if-usefulpossible)
* [ ] [Reviewed for breaking changes](https://developer.matomo.org/guides/pull-request-reviews#reviewed-for-breaking-changes)
* [ ] [Developer changelog updated if needed](https://developer.matomo.org/guides/pull-request-reviews#developer-changelog-updated-if-needed)
* [ ] [Documentation added if needed](https://developer.matomo.org/guides/pull-request-reviews#documentation-added-if-needed)
* [ ] Existing documentation updated if needed

### CODE REVIEW

File: docs/4.x/spa-tracking.md [hunk 1]
```diff
@@ -2,9 +2,9 @@
 category: Integrate
 ---
 
-# Single-Page Application Tracking
+# Single-Page Application/Progressive Web App Tracking
 
-Single-page websites and web applications have become a standard over the last years. Getting the tracking of such websites and apps right is crucial to your success as you need to ensure the measured data is meaningful and correct.
+Single-page websites and Progressive Web App have become a standard over the last years. Getting the tracking of such websites and apps right is crucial to your success as you need to ensure the measured data is meaningful and correct.
```
Comment (Findus23) [inline docs/4.x/spa-tracking.md]:
```suggestion
Single-page websites and Progressive Web Apps have become a standard over the last years. Getting the tracking of such websites and apps right is crucial to your success as you need to ensure the measured data is meaningful and correct.
```

File: docs/4.x/spa-tracking.md [hunk 2]
```diff
@@ -143,6 +143,9 @@ To trigger your Matomo tag (which calls `trackPageView`), you can either:
 2. or in your Single Page App, if you are using the 'Pageview Trigger' to trigger a Pageview, you can trigger a Tag Manager Event `{event: 'mtm.PageView'}` by calling the following line in JavaScript: `_mtm.push({'event': 'mtm.PageView'});`. 
    - This would also work similarly when you use instead the 'DOM Ready Trigger' (call `_mtm.push({'event': 'DOMReady'});`) or when you use the 'Window Loaded Trigger' (call `_mtm.push({'event': 'WindowLoad'});`
 
+## Offline Tracking
+
+If your web application require offline tracking, refer [Matomo Offline Tracking](https://matomo.org/faq/how-to/how-do-i-set-up-matomo-offline-tracking/)
```
Comment (Findus23) [inline docs/4.x/spa-tracking.md]:
```suggestion
If your web application requires offline tracking, please refer to the [Matomo  offline tracking](https://matomo.org/faq/how-to/how-do-i-set-up-matomo-offline-tracking/) guide.
```

**Review by Findus23 — CHANGES_REQUESTED** [review]
Just some minor text changes.

File: docs/4.x/spa-tracking.md [hunk 3]
```diff
@@ -143,6 +143,9 @@ To trigger your Matomo tag (which calls `trackPageView`), you can either:
 2. or in your Single Page App, if you are using the 'Pageview Trigger' to trigger a Pageview, you can trigger a Tag Manager Event `{event: 'mtm.PageView'}` by calling the following line in JavaScript: `_mtm.push({'event': 'mtm.PageView'});`. 
    - This would also work similarly when you use instead the 'DOM Ready Trigger' (call `_mtm.push({'event': 'DOMReady'});`) or when you use the 'Window Loaded Trigger' (call `_mtm.push({'event': 'WindowLoad'});`
 
+## Offline Tracking
+
+If your web application requires offline tracking, please refer to the [Matomo  offline tracking](https://matomo.org/faq/how-to/how-do-i-set-up-matomo-offline-tracking/) guide.
```
Comment (sgiehl) [inline docs/4.x/spa-tracking.md]:
```suggestion
If your web application requires offline tracking, please refer to the [Matomo offline tracking](https://matomo.org/faq/how-to/how-do-i-set-up-matomo-offline-tracking/) guide.
```

**Pull request merged**